pkgs/by-name/im/imhex/package.nix

```diff
@@ -60,29 +78,41 @@ stdenv.mkDerivation rec {
     mbedtls
     nlohmann_json
     yara
+  ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk_11_0.frameworks.UniformTypeIdentifiers
   ];
 
   cmakeFlags = [
-    "-DIMHEX_OFFLINE_BUILD=ON"
-    "-DUSE_SYSTEM_CAPSTONE=ON"
-    "-DUSE_SYSTEM_CURL=ON"
-    "-DUSE_SYSTEM_FMT=ON"
-    "-DUSE_SYSTEM_LLVM=ON"
-    "-DUSE_SYSTEM_NLOHMANN_JSON=ON"
-    "-DUSE_SYSTEM_YARA=ON"
+    (lib.cmakeBool "IMHEX_OFFLINE_BUILD" true)
+    (lib.cmakeBool "IMHEX_COMPRESS_DEBUG_INFO" false) # avoids error: cannot compress debug sections (zstd not enabled)
+    (lib.cmakeBool "IMHEX_GENERATE_PACKAGE" stdenv.isDarwin or stdenv.isWindows)
+    (lib.cmakeBool "USE_SYSTEM_CAPSTONE" true)
+    (lib.cmakeBool "USE_SYSTEM_CURL" true)
+    (lib.cmakeBool "USE_SYSTEM_FMT" true)
+    (lib.cmakeBool "USE_SYSTEM_LLVM" true)
+    (lib.cmakeBool "USE_SYSTEM_NLOHMANN_JSON" true)
+    (lib.cmakeBool "USE_SYSTEM_YARA" true)
   ];
 
-  # rsync is used here so we can not copy the _schema.json files
   postInstall = ''
     mkdir -p $out/share/imhex
     rsync -av --exclude="*_schema.json" ${patterns_src}/{constants,encodings,includes,magic,patterns} $out/share/imhex
```

Comment:
```suggestion
    rsync -av --exclude="*_schema.json" ${patterns_src}/{constants,encodings,includes,magic,nodes,patterns} $out/share/imhex
```

Reply:
What can we do to avoid use of rsync? It feels unnecessary...

Reply:
Great! That matches the answers I got from the author. Thank you. 
Is the share folder relevant on Darwin or should we only do that on Linux?

Reply:
`find ... -exec cp {} target/` is always an option. I agree that rsync should not be a dependency for a package build unless its incremental transfers actually provide a speedup/reduce network traffic.

Reply:
IMO `rsync` is fine here, it’s just a souped‐up `cp` in many ways and it doesn’t have many dependencies. No objections to replacing it with `find`, though.